src/main/java/uk/gov/justice/digital/services/RedShiftTableExpiryService.java

```diff
@@ -20,7 +20,7 @@ public class RedShiftTableExpiryService {
             "SELECT tablename " +
             "FROM SVV_EXTERNAL_TABLES " +
             "WHERE schemaname = 'reports' " +
-            "AND json_extract_path_text(parameters, 'transient_lastDdlTime', TRUE) < (EXTRACT(EPOCH FROM GETDATE()) - %d)";
+            "AND COALESCE(json_extract_path_text(parameters, 'transient_lastDdlTime', TRUE), 0) < (EXTRACT(EPOCH FROM GETDATE()) - %d)";
```

Comment:
I tried running this on Redshift 
```
SELECT tablename
FROM SVV_EXTERNAL_TABLES 
WHERE schemaname = 'reports' 
AND COALESCE(json_extract_path_text(parameters, 'transient_lastDdlTime', TRUE), 0) < (EXTRACT(EPOCH FROM GETDATE()) - 3600)
```
and I get:
`ERROR: COALESCE types text and integer cannot be matched `

Reply:
Also running this query in Dev:
```
SELECT tablename,json_extract_path_text(parameters, 'transient_lastDdlTime', TRUE)
FROM SVV_EXTERNAL_TABLES 
WHERE schemaname = 'reports' 
```
returns null in all rows for the second column.